src/votekit/cvr_loaders.py

```diff
@@ -68,3 +69,89 @@ def rank_column_csv(
+    cands_included = False
+
+    if not os.path.isfile(fpath):
+        raise FileNotFoundError(f"File with path {fpath} cannot be found")
+    if os.path.getsize(fpath) == 0:
+        raise EmptyDataError("Dataset cannot be empty")
+
+    with open(fpath, "r") as file:
+        for i, line in enumerate(file):
+            s = line.rstrip("\n").rstrip()
+            if i == 0:
+                # first number is number of candidates, second is number of seats to elect
+                metadata = [int(data) for data in s.split(" ")]
+                if len(metadata) != 2:
+                    raise DataError(
+                        "metadata (first line) should have two parameters"
+                        " (number of candidates, number of seats)"
+                    )
+                seats = metadata[1]
+            # read in ballots, cleaning out rankings labeled '0' (designating end of line)
+            if numbers:
+                ballot = [int(vote) for vote in s.split(" ")]
+                num_votes = ballot[0]
+                # ballots terminate with a single row with the character '0'
+                if num_votes == 0:
+                    numbers = False
+                else:
+                    ranking = [rank for rank in list(ballot[1:]) if rank != 0]
+                    b = (ranking, num_votes)
+                    ballots.append(b)  # this is converted to the PP format later
+            # read in candidates
+            elif "(" in s:
+                cands_included = True
+                name_parts = s.strip('"').split(" ")
+                first_name = " ".join(name_parts[:-2])
+                last_name = name_parts[-2]
+                party = name_parts[-1].strip("(").strip(")")
+                names.append(str((first_name, last_name, party)))
+            else:
+                if len(names) != metadata[0]:
+                    err_message = (
+                        f"Number of candidates listed, {len(names)}," + f" differs from"
+                        f"number of candidates recorded in metadata, {metadata[0]}"
+                    )
+                    raise DataError(err_message)
+                # read in election location (do we need this?)
+                # location = s.strip("\"")
+                if not cands_included:
+                    raise DataError("Candidates missing from file")
+                # map candidate numbers onto their names and convert ballots to PP format
+                for i, name in enumerate(names):
+                    name_map[i + 1] = name
+                ballots = [
+                    Ballot(
+                        ranking=[{name_map[cand]} for cand in ballot[0]],
+                        weight=Fraction(ballot[1]),
+                    )
+                    for ballot in ballots
+                ]
+
+                return PreferenceProfile(ballots=ballots, candidates=names), seats
+    raise Exception(f"Undetermined error parsing file '{fpath}'.")
```

Comment:
this is probably redundant given the thoroughness of your other checks + python's built in errors will probably be raised during parsing before this gets triggered

Reply:
ok, I think I removed it? but not sure if this got connected to this PR